src/modules/common/components/resources-list.jsx

```diff
@@ -52,7 +51,7 @@ const ResourcesList = ({
                   <small>{`by ${resource.links.owner.display_name}`}</small>}
                 {showStatus &&
                   <span
-                    className={`color-label ${resource.launch_approved ? 'green' : 'red'}`}
+                    className={`resources-list__status color-label ${resource.launch_approved ? 'green' : 'red'}`}
```

Comment:
FYI, `classnames` is an installed dependency of the project and is really handy for setting conditional classes. 

Also, using BEM convention, it should be `color-label--red` or `color-label--green`

Out of scope of this PR though.